IABot/tests/acceptance/test_cases/acceptance-result.txt

```diff
@@ -0,0 +1,17 @@
+Some text.
+<ref name="raw-live-url">https://en.wikipedia.org/wiki/Main_Page</ref>
+<ref name="raw-dead-url">{{cite web|url=http://www.statistics.sk/mosmis/eng/run.html |title=Archived copy |accessdate=2007-05-13 |deadurl=yes |archiveurl=https://web.archive.org/web/20070513023228/http://www.statistics.sk/mosmis/eng/run.html |archivedate=2007-05-28 |df= }}</ref>
+<ref name="raw-dead-url-w-text">Doe, Jane (1977). http://www.statistics.sk/mosmis/eng/run.html.{{dead link}} More text.</ref>
```

Comment:
Oh right, that period.  It stops grabbing the contents of a plain URL if it's not a whitespace.